lib/modules/manager/regex/utils.ts

```diff
@@ -97,3 +97,10 @@ export function mergeExtractionTemplate(
     replaceString: addition.replaceString ?? base.replaceString,
   };
 }
+
+export function isValidDependency(dep: PackageDependency): boolean {
+  // check if all the fields are set
+  return !['depName', 'currentValue'].some(
+    (value) => !dep[value as keyof PackageDependency]
+  );
```

Comment:
can't we directly check the two fields, that code looks too ugly and hides type checks.

Reply:
I have changed it to direct checks, but my thought has been that at some point there will be a list of minimal values which could then replace the static array.